cmd/operator/main.go

```diff
@@ -54,19 +55,46 @@ func init() {
 
 type devicePluginControllerAndWebhook map[string](func(ctrl.Manager, string, bool) error)
 
+type flagList []string
+
+func (fl *flagList) String() string {
+	return strings.Join(*fl, ", ")
+}
+
+func (fl *flagList) Set(value string) error {
+	*fl = append(*fl, value)
+	return nil
+}
+
+func enabled(arr []string, val string) bool {
+	for _, s := range arr {
```

Comment:
nit: if we change flagList []string => flagMap map[string]bool we can avoid iterating over the list of plugins.
this function would look something like this:
```
func (fm *flagMap) enabled(key string) bool {
    return (*fm)[key]
}
```

Reply:
OK, good idea, thanks!

Reply:
@ozhuraki can you make this change still?

Reply:
@mythi 

We tried to do it with map as @bart0sh suggested. However we later looked at it, @bart0sh changed his mind and we changed it back to the list, since the list approach looked slightly simpler. It's not principal, I can still change, shall we?

Reply:
@ozhuraki I had missed that change and this conversation was still open. I'm OK with this approach too but I'm not sure how it's simpler :-)